Docker/LocalSettings.php.template

```diff
@@ -54,3 +54,7 @@ require_once "${DOLLAR}IP/extensions/Wikibase/repo/Wikibase.php";
 require_once "${DOLLAR}IP/extensions/Wikibase/repo/ExampleSettings.php";
 require_once "${DOLLAR}IP/extensions/Wikibase/client/WikibaseClient.php";
 require_once "${DOLLAR}IP/extensions/Wikibase/client/ExampleSettings.php";
+
+if( isset(${DOLLAR}_ENV["DOCKER_INCLUDE_SETTINGS"]) && ${DOLLAR}_ENV["DOCKER_INCLUDE_SETTINGS"] != '' ) {
```

Comment:
this might be a bit controversial, we could also copy this settings file (which is the one we ship in docker) and override it rather than extending it.